prompt2model_demo.py

```diff
@@ -351,46 +353,110 @@ def main():
         evaluator_logger = get_formatted_logger("ModelEvaluator")
         evaluator_logger.setLevel(logging.INFO)
 
+        train_batch_size = None
+
         while True:
-            line = input("Enter the training batch size:")
+            line = input(
+                "Are you interested to train the model with automatic hyperparameter search? Type 'y' for Yes and 'n' for No. "  # noqa E501
+            )
             try:
-                train_batch_size = int(line)
-                assert 0 < train_batch_size
+                assert line in ["y", "n"]
                 break
             except Exception:
-                line_print("The training batch size must be greater than 0.")
+                line_print("The answer should be either y or n")
         time.sleep(1)
 
-        while True:
-            line = input("Enter the number of epochs to train for:")
-            try:
-                num_epochs = int(line)
-                break
-            except ValueError:
-                line_print("Invalid input. Please enter a number.")
-        time.sleep(1)
+        if line == "y":
+            line_print("Moving with training and hyperparameter selection.")
+            max_num_epochs = input(
+                "Enter max number of epochs: Press enter to use default value: "
+            )
+            num_trials = input(
+                "Enter the number of trials to conduct hypeparamter search. Press enter to use default value: "  # noqa E501
+            )
+            max_batch_size = input(
+                "Enter the max batch size Press enter to use default: "
+            )
```

Comment:
Suggestion to add punctuation + specify what the current default is.

```suggestion
            default_batch_size = DEFAULT_HYPERPARAMETERS_SPACE["per_device_train_batch_size"][0]  # noqa E501
            max_batch_size = input(
                "Enter the max batch size. " + \
                f"Press enter to use default: {default_batch_size}"
            )
```

Reply:
If you accept my suggestion above to change the default hyperparameter space setting of `per_device_train_batch_size` from `[2, 4]` (a list) to `2` (a constant), then change line 377 above to:
```
            default_batch_size = DEFAULT_HYPERPARAMETERS_SPACE["per_device_train_batch_size"]  # noqa E501
```

Reply:
Yeah, I overlooked this one. Fixed now